procedures/foreman/release.md.erb

```diff
@@ -1,15 +1,15 @@
-# Writing release notes
+# Manual updates
 
 * [ ] Update manual if applicable for any additional installation steps
-* [ ] Draft release notes in markdown ([example](https://gist.github.com/domcleal/5567450)), with these sections (and do not use personal pronouns):
-  * [ ] Headline features: half a dozen important features with a few sentences description each
-  * [ ] Upgrade notes: all important notices that users must be aware of before upgrading
+* Update [release notes section](gh-pages/_includes/manuals/<%= short_version %>/1.2_release_notes.md) in the manual:
   * [ ] Release notes: bullet point list by category of all changes, include link to bug numbers. You can auto-generate changes using the [release notes script](https://github.com/theforeman/theforeman.org/blob/gh-pages/scripts/release_notes.rb).
   * [ ] CLI release notes are taken from the hammer-cli and hammer-cli-foreman changelogs
   * [ ] Link to installer changelogs and note versions being used
-  * [ ] Update installer options section using the [get-params script](https://github.com/theforeman/theforeman.org/blob/gh-pages/scripts/installer/get-params)
-* [ ] Get the apipie doc and place it in [api](https://github.com/theforeman/theforeman.org/tree/gh-pages/api)/<%= short_version %>
-
+<% if (is_rc || full_version.end_with?('0')) %>  * [ ] Headline features: half a dozen important features with a few sentences description each
+  * [ ] Upgrade notes: all important notices that users must be aware of before upgrading
+* [ ] Update installer options section using the [get-params script](https://github.com/theforeman/theforeman.org/blob/gh-pages/scripts/installer/get-params)
```

Comment:
I'd actually say this should always be done. Ideally it would be no difference, but otherwise you know.

Reply:
specifically the installer options or all of them?

Reply:
I'd prefer both installer params and apipie. Ideally it should be no diff (though I think apipie isn't that stable).

Reply:
apipie is actually pretty stable, and shouldn't change in .z versions normally. Also generating the docs takes a couple of hours on my computer, so i'd prefer not to redo it every time. I'll update this though to remind to check that in z stream.

Reply:
In meant in the generated HTML. If you rerun the same job twice, it may change formatting I think due to changing examples.

Reply:
During 1.23 we noticed the script only works on already released bits. Probably good to include in the instructions somewhere.

Reply:
added comment about that